### PR TITLE
UPM meta updated

### PR DESCRIPTION
**Pull request by semuserable**
closes #124

* renamed `LICENSE` to `LICENSE.md` based on [Unity layout docs](https://docs.unity3d.com/Manual/cus-layout.html)
* `Documentation~` added
* CI commands updated

### CODE REVIEW

**Comment by semuserable** [comment]
Tbh, I dunno how to check that it's got correctly packaged in the CI (`ci.yaml`) commands. If someone can provide a path, I'd appreciate it.

File: package/Documentation~/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Sentry SDK for Unity
+
+> Use [official documentation](https://docs.sentry.io/platforms/unity/) for more information.
+
+Start using it by opening `Tools` -> `Sentry` from `Unity` editor.
```
Comment (bruno-garcia) [inline package/Documentation~/index.md:5]:
```suggestion
Start using it by opening `Tools` -> `Sentry` in the `Unity` editor.

```

File: package/Documentation~/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Sentry SDK for Unity
+
+> Use [official documentation](https://docs.sentry.io/platforms/unity/) for more information.
+
+Start using it by opening `Tools` -> `Sentry` from `Unity` editor.
+
+Minimal configuration:
+
+* `DSN` - input your [Sentry DSN](https://docs.sentry.io/product/sentry-basics/dsn-explainer/) value
```
Comment (bruno-garcia) [inline package/Documentation~/index.md:9]:
```suggestion
* `DSN` - Get your [Sentry DSN](https://docs.sentry.io/product/sentry-basics/dsn-explainer/) when you create a new project in Sentry. Or in the project settings.

```

File: package/Documentation~/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+> Use [official documentation](https://docs.sentry.io/platforms/unity/) for more information.
+
+Start using it by opening `Tools` -> `Sentry` from `Unity` editor.
+
+Minimal configuration:
+
+* `DSN` - input your [Sentry DSN](https://docs.sentry.io/product/sentry-basics/dsn-explainer/) value
+
+Create `MonBehaviour` and assign it to any object you have in the scene, press play.
```
Comment (bruno-garcia) [inline package/Documentation~/index.md:11]:
```suggestion
### Quick example

[Sentry's the documentation](https://docs.sentry.io/platforms/unity) includes **much** more you can do, including performance monitoring, adding tags, and more. But to get you started, here's a simple example using Unity's `Debug.LogError`:

Create `MonoBehaviour` and assign it to any object you have in the scene, press play.
```

**Comment by semuserable** [comment]
Suggestions in.

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**